.gitignore

```diff
@@ -43,3 +43,6 @@ event.json
 
 # Nx caching
 nx-cache
+
+# Build cache manifests
+*.tsbuildinfo
```

Comment:
I'm also wondering whether this should be cached by Nx as well. In this case, it should be included in the root Nx config

Reply:
You are completely right 😉